File: ClasesLuis/Paquete3/Clase23/explicaciones.py
def suma(numero1,numero2):
    print(numero1+numero2)
def resta(numero1,numero2):
    print(numero1-numero2)
def multiplicacion(numero1,numero2):
    print(numero1*numero2)
def division(numero1,numero2):
    if (numero2 == 0):
        error()
    else:
        print(numero1/numero2)
def error():
    print("Numero no válido")

while(True):
    pregunta=int(input("Que quieres hacer? 1= sumar 2= restar 3= multiplicar 4=dividir 5=salir : "))
    if (pregunta == 5):
        print("Adios")
        break
    if (pregunta > 5 and pregunta <= 0):
        error()
    else:
        numero1=float(input("Hola, introduzca un numero: "))
        numero2=float(input("Hola, introduzca otro numero: "))
        if(pregunta == 1):
            suma(numero1,numero2)
        elif(pregunta == 2):
            resta(numero1,numero2)
        elif(pregunta == 3):
            multiplicacion(numero1,numero2)
        elif(pregunta == 4):
            division(numero1,numero2)


        

File: ClasesMigue/Clases/src/clase2/explicaciones.java
package clase2;

import java.util.Scanner;

public class explicaciones {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Escribe el primer número: ");
		int numero1 = sc.nextInt();
		
		System.out.print("Escribe el segundo número: ");
		int numero2 = sc.nextInt();
		
		System.out.println("Seleccione una operacion: ");
		System.out.println("1) Sumar");
		System.out.println("2) Restar");
		System.out.println("3) Multiplicar");
		System.out.println("4) Dividir");
		
		int eleccion = sc.nextInt();
		
		if (eleccion == 1) {
			int suma = numero1 + numero2;
			System.out.println("La suma es: " + suma);
		} else if (eleccion == 2){
			int resta = numero1 - numero2;
			System.out.println("La resta es: " + resta);
		} else if (eleccion == 3){
			int multiplicacion = numero1 * numero2;
			System.out.println("La multiplicacion es: " + multiplicacion);
		} else if (eleccion == 4){
			int division = numero1 / numero2;
			System.out.println("La division es: " + division);
		} else {
			System.out.println("Numero no válido");
		}
		
		
		

		

	}

}
